lib/spreadsheet_exporter/spreadsheet.rb

```diff
@@ -28,14 +47,14 @@ def self.from_objects(objects, options = {})
 
     # Return an array of human_attribute_name's
     # Used by the CSV Import/Export process to match CSV headers to model attribute names
-    def self.han(klass, *attributes)
-      options = attributes.extract_options!
+    def self.han(headers, humanize_headers_class:, downcase: false, **)
```

Comment:
The return type for this use to be scalar or array but now is always an array.  This change didn't break any other code that I could see